tools/kagami/src/main.rs

```diff
@@ -849,13 +853,69 @@ mod validator {
         }
     }
 
+    pub fn clone_default_validator(out_dir: impl AsRef<Path>) -> Outcome {
+        const GIT_ORIGIN: &str = "https://github.com/hyperledger/iroha.git";
```

Comment:
Same const is duplicated in `kagami::swarm`. We should probably move all meta consts in a single place.

Reply:
Got rid of this func entirely, but didn't really refactor out the consts and common functionality yet.